### PR TITLE
Removed bond length constraints on atoms objects before structures co…

### PR DESCRIPTION
**Pull request by tdprice-858**
…mparison. Reapplied fix atom constraints to slab.

### CODE REVIEW

File: pynta/utils.py [hunk 1]
```diff
@@ -113,6 +114,11 @@ def get_unique_sym_struct_indices(geoms):
 
     for i,geom in enumerate(geoms_copy):
         adsorbate_atom_obj = geom
+        for constraint in adsorbate_atom_obj.constraints:
+            if isinstance(constraint, FixAtoms):
+                c = constraint # Capturing constraints of slab
+        adsorbate_atom_obj.set_constraint() # Reset constraints
+        adsorbate_atom_obj.set_constraint(c) # Apply slab constraints
```
Comment (mjohnson541) [inline pynta/utils.py:121]: Why do we want the slab constraints to be applied?

File: pynta/utils.py [hunk 2]
```diff
@@ -113,6 +114,11 @@ def get_unique_sym_struct_indices(geoms):
 
     for i,geom in enumerate(geoms_copy):
         adsorbate_atom_obj = geom
+        for constraint in adsorbate_atom_obj.constraints:
```
Comment (mjohnson541) [inline pynta/utils.py:117]: We do this here for only one of the structure comparison functions, but shouldn't we have this in all of them?

File: pynta/utils.py (hunk 1 above)
Comment (tdprice-858) [reply pynta/utils.py:121]: I don't know if we do want slab constraints applied. I just noticed that having slab constraints did not lead to a convergence error.
I can look into this further and see if we should just remove them all

File: pynta/utils.py (hunk 2 above)
Comment (tdprice-858) [reply pynta/utils.py:117]: Yeah probably. I can adjust them all after I figure out whether or not we want to add slab constraints before comparison.

**Pull request closed without merging**